src/components/capture/useDiscoverCapture.ts

```diff
@@ -248,7 +150,11 @@ function useDiscoverCapture(
                     })
                     .order('created_at', { ascending: false }),
                 async (payload: any) => {
-                    await storeDiscoveredCollections(payload.draft_id);
+                    await storeDiscoveredCollections(
+                        payload.draft_id,
+                        entityType,
+                        callFailed
+                    );
```

Comment:
Passing these props just because they are required now since this is being created with a hook. Instead of passing these into the hook I decided to pass to the function to reduce the number of "memo" runs would be needed.